### ISSUE TITLE
centos URL used for RedHat8, Rocky8, and Alma8

### ISSUE THREAD

**Issue by jbagleyjr** [puppetlabs/puppetlabs-docker#998]
## Describe the Bug
Incorrect URL is used for the package_ce_source_location that includes "/centos/" which the docker stopped updating ~5 months ago.

## Expected Behavior
Latest docker-ce package should be installed but were getting an older version because the incorrect repo URL is used.

## Steps to Reproduce
Apply on any Rocky8, Alma8 or RedHat8.  Docker version installed is 26 but should be 27.

## Environment
 - Platform Rocky8 mostly, but we also have some redhat8.

## Additional Context
Compare these two repos:
https://download.docker.com/linux/rhel/8/x86_64/stable/Packages/
https://download.docker.com/linux/centos/8/x86_64/stable/Packages/

Seems like the rhel repo would be appropriate for our rocky and alma servers.

**Comment by bastelfreak** [comment]
Hi, I assume the problem is that CentOS 8 is EoL, so they don't update it anymore. For Rocky/Alma, they should use the rhel dir, not centos. Are you able to provide a PR for this?

**Comment by jbagleyjr** [comment]
Created pull request https://github.com/puppetlabs/puppetlabs-docker/pull/999 and am testing this fix in our environment.

**Comment by jbagleyjr** [comment]
confirmed the fix.  We'll use my fork in our Puppetfile until the pull request is merged.

**Comment by uoe-pjackson** [comment]
Looks like pull request #999  is blocked by CLA. The latest version of docker in the centos folder is 26.1.3 and the 26.1.x family went End of Life on 17 February 2025. I've mentioned jbagleyjr in the hope that they can resolve the CLA check. Given it's a one liner and has become a higher priority I happy to create a new PR if the CLA issue can't be resolved soon.

**Comment by jbagleyjr** [comment]
Sorry for the delay on my pull request.  I had sent the CLA to my legal department and never heard back.  I've signed it today for myself personally and not for the company I work for.
